pkg/controllers/csi/metadata/sqlite.go

```diff
@@ -304,7 +277,7 @@ func (access *SqliteAccess) executeAlterStatement(ctx context.Context, statement
 	return nil
 }
 
-// Setup connects to the database and creates the necessary tables if they don't exist
+// setup connects to the database and creates the necessary tables if they don't exist
```

Comment:
why?

Reply:
that's strange- didn't do it on purpose- I suppose it's a refactoring gone wrong by goland